File: JAVA VW/src/programowanie/obiektowe/proj/Grass.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package programowanie.obiektowe.proj;

/**
 *
 * @author kubaz
 */
public class Grass extends Plant{
    
        public Grass(Position position, World world) {
        super(position, world);
        this.name="Grass";
        this.initiative=0;
        this.strength=0;
    }   
}

File: CPP VW/CPP VW/Grass.cpp
#include "Grass.h"

Grass::Grass(World& world, int x, int y) :
	Plant(world, x, y)
{
	this->sign = 'g';
	this->power = 0;
	this->initiative = 0;
}
bool Grass::specialCollision(Organism* other) {
	return false;
}
bool Grass::specialAction() {
	return false;
}

Grass::~Grass(){
}
